extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-postgresql/src/main/java/org/apache/guacamole/auth/postgresql/conf/PostgreSQLEnvironment.java

```diff
@@ -48,6 +48,22 @@
      */
     private static final int DEFAULT_PORT = 5432;
 
+    /**
+     * The default number of seconds the driver will wait for a response from
+     * the database. A value of 0 (the default) means the timeout is disabled.
+     */
+    private static final int DEFAULT_STATEMENT_TIMEOUT = 0;
+
+    /**
+     * The default timeout (in seconds) used for socket read operations.
+     * If reading from the server takes longer than this value, the
+     * connection is closed. This can be used to handle network problems
+     * such as a dropped connection to the database. Similar to 
+     * DEFAULT_STATEMENT_TIMEOUT, it will also abort queries that take too 
```

Comment:
Two comments, here:
* You say "similar to DEFAULT_STATEMENT_TIMEOUT", but there's no mention of aborting queries on the above timeout.
* It looks like most of the documentation around what these two additional parameters do is here in the `DEFAULT_` sections, rather than down in the `PostgreSQLGuacamoleProperties.java` file.  If you look at the documentation of other `DEFAULT_`s around this, you'll see that the documentation is more succinct and references the actual properties.  I suggest doing the same with these two.